### DISCUSSION TITLE
Tailscale setup error

### DISCUSSION THREAD

**Discussion by salilwalavalkar** [tteck/Proxmox#2100] [Help Needed] [answered]
Proxmox 8.0.4 installed. I am able to install other containers like ubuntu, home assistant etc. The tailscale one always fails. Any ideas?

<img width="682" alt="ts-setup" src="https://github.com/tteck/Proxmox/assets/1934828/c940c7a8-e898-4be6-b51b-d3d635138e0b">

**Comment by tteck** [comment]
Are you executing the script within the Proxmox shell?

**Reply by salilwalavalkar** [reply]
Yes, I am
<img width="773" alt="rt" src="https://github.com/tteck/Proxmox/assets/1934828/4bbac852-05cb-40ec-9ff9-e0216abd163f">

**Reply by salilwalavalkar** [reply]
Picked the script from helper scripts

**Reply by tteck** [reply]
No issues here

![Screenshot 2023-11-25 1 53 32 PM](https://github.com/tteck/Proxmox/assets/21342670/c21f06d0-1e9f-4ecd-b279-7f0747f1e752)

**Reply by salilwalavalkar** [reply]
Running below script

`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/add-tailscale-lxc.sh)"`

**Reply by tteck** [reply]
yep, me too

**Reply by tteck** [reply]
Let me take a look at the script... One sec

**Reply by salilwalavalkar** [reply]
Just tried Promethus one to test..it worked

`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/prometheus.sh)"`

**Reply by salilwalavalkar** [reply]
I have installed with success
1.  Ubuntu LXC  
2. Home assistant LXC

The issue is only with Tailscale.

**Reply by tteck** [reply]
please try again (clearing browser cache may be required)

**Reply by salilwalavalkar** [reply]
Great! I moved to next step now 👍 
<img width="627" alt="ts-2" src="https://github.com/tteck/Proxmox/assets/1934828/8c97e01d-440e-4448-9c2c-0b887e0c6898">

**Reply by salilwalavalkar** [reply]
Interesting that there are no VMs listed

**Reply by tteck** [reply]
Do you have any LXCs created?

**Reply by salilwalavalkar** [reply]
Yes I have these 2 
https://github.com/tteck/Proxmox/discussions/2100#discussioncomment-7668053

**Reply by tteck** [reply]
What does the output from this command show?
```
pct list | awk 'NR>1'
```

**Reply by salilwalavalkar** [reply]
Blank. Even `pct list` returns nothing

**Reply by tteck** [reply]
Are you running multiple nodes?

**Reply by salilwalavalkar** [reply]
Sorry did not get the question. It's just one physical machine which is running proxmox. 
Using your scripts, I have installed https://github.com/tteck/Proxmox/discussions/2100#discussioncomment-7668053

**Reply by tteck** [reply]
Why doesn't `pct list` show anything? You have to find the answer to that.

**Reply by tteck** [reply]
Try rebooting Proxmox.

**Reply by salilwalavalkar** [reply]
Sure, give me some time to figure it out. Appreciate your help.

**Reply by salilwalavalkar** [reply, marked as answer]
@tteck Sorry for the delay; I found out why I am getting no output from the `pct list`. I have installed virtual machines and not LXC containers. I get output when I type `qm ist`. I did use your scripts to install both of them https://tteck.github.io/Proxmox/

<img width="321" alt="qm" src="https://github.com/tteck/Proxmox/assets/1934828/a5d6308a-3d30-4dea-a45c-25dc7070e218">

**Reply by tteck** [reply]
The Tailscale script is only for LXCs.
